web/server.go

```diff
@@ -116,6 +125,18 @@ func (s Server) serveHTTP(w http.ResponseWriter, r *http.Request) {
 	renderError(r.Context(), w, APIErrNotFound)
 }
 
+func (s Server) serveHTTPS(w http.ResponseWriter, r *http.Request) {
+	if strings.HasPrefix(r.URL.Path, "/api/v1/") {
+		s.handleAPIV1HTTPS(w, r)
+		return
+	}
+	if r.URL.Path == "/test" {
+		w.Write([]byte("response from sabakan TLS\n"))
+		return
+	}
```

Comment:
Whould you like to keep this?

Reply:
Sorry, I forgot to delete this test API. Thank you.